docs/usage.rst

```diff
@@ -284,6 +283,18 @@ Recall the ordering of the players::
 
 Thus we see that there are multiple Nash equilibria for this game. Two pure equilibria that involve both players playing :code:`Defector` and both players playing :code:`TitForTat`.
 
+The ResultSet object also includes the 'wins' attribute, which is derived from the payoff matrix and lists the number of wins for each player::
+
+  import axelrod
+  strategies = [s() for s in axelrod.demo_strategies]
+  tournament = axelrod.Tournament(strategies)
+  results = tournament.play()
+  results.wins
+
+The resulting wins list is::
+
+  [0, 80, 0, 20, 30]
```

Comment:
Could you add a quick sentence saying something like 'which shows that Cooperator had 0 wins and Defector 80 wins' (just to point at the order that we get out)?